src/models/Text.ts

```diff
@@ -0,0 +1,33 @@
+import { types, Instance, SnapshotIn } from 'mobx-state-tree';
+
+import { EntityType } from 'src/types/entity';
+import { blockAliases } from 'src/aliases';
+
+const Text = types.model({
+	id: types.identifier,
+	type: types.literal('text'),
+	params: types.model({
+		x: types.number,
+		y: types.number,
+		isSelected: true,
+		copy: types.model({
+			en: "",
```

Comment:
Set the default as "Text" so it appears like that when you add a new one. :wink:

Reply:
fixed